cypress/components/decimal/decimal.cy.tsx

```diff
@@ -238,6 +239,27 @@ context("Tests for Decimal component", () => {
       }
     );
 
+    it.each(testData)(
+      "should render Decimal with prefix prop set to %s",
+      (prefix) => {
+        CypressMountWithProviders(<Decimal prefix={prefix} />);
+
+        textboxPrefix()
+          .should("have.text", prefix)
+          .and("have.css", "font-size", "14px")
+          .and("have.css", "font-weight", "900")
+          .and("have.css", "margin-left", "12px");
+      }
+    );
+
+    it("when prefix prop is set, 'flex-direction' should be 'row'", () => {
```

Comment:
nitpick(non-blocking): tests should read in plain English ideally

This is something that is common across the project, so I wouldn't worry about changing it but something to keep in mind potentially. 

```suggestion
    it("should set `flex-direction` to `row` when a prefix prop is used", () => {
```